pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -127,21 +127,22 @@ async def run(self):
                     pr = await self._create_or_update_pull_request(nvrs)
 
             # Sends a slack message
-            message = f"Hi @release-artists , microshift for assembly {self.assembly} has been successfully built."
+            message = f"microshift for assembly {self.assembly} has been successfully built."
             if pr:
                 message += f"\nA PR to update the assembly definition has been created/updated: {pr.html_url}"
-                message += "\nReview and merge the PR before you proceed.\n"
-            message += '\n'.join([
-                "To attach the build to Errata, run <https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fprepare-release/|prepare-release> job, or:",
-                "```",
-                f"elliott --group {self.group} --assembly {self.assembly} --rpms microshift find-builds -k rpm --member-only --use-default-advisory microshift",
-                f"elliott --group {self.group} --assembly {self.assembly} attach-cve-flaws --use-default-advisory microshift",
-                "```"
-            ])
             if assembly_type in [AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE]:
                 message += f"\n This is a {assembly_type.name} release. Please run <https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fmicroshift_sync/|microshift_sync> to publish the build to mirror."
             if slack_client:
                 await slack_client.say(message, slack_thread)
+            # prepare microshift advisory in a new slack thread
+            if pr and slack_client:
+                slack_thread await slack_client.say(f"Start preparing microshift advisory for assembly {self.assembly}.")
+                await self._attach_builds()
+                await self._sweep_bugs()
+                await self._attach_cve_flaws()
+                await self._verify_microshift_bugs()
+                await self._change_advisory_status()
```

Comment:
Now they will raise an exception on failure. You can try-except them and print out the error message and instructions that you have removed.

Reply:
errors in exectools will print the failed cmd, and since we treat the exception as failed now, release artist will debug the issue to check what's wrong

Reply:
maybe exectools should add a parameter to support customized exception error message, we can improve that later

Reply:
OK, I kinda agree to ask release-artists to go the exception message to figure out why an error happens.

Reply:
> maybe exectools should add a parameter to support customized exception error message, we can improve that later 

I don't think so. The raw exception message can help release-artist to troubleshoot, however a custom error message might hide it.